packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Fragment,
+  Prop,
+  h,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-telephone',
+  styleUrl: 'va-telephone.css',
+  shadow: true,
+})
+export class VaTelephone {
+  /**
+   * 3 or 10 digit string representing the contact number
+   */
+  @Prop() contact!: string;
```

Comment:
The `!` means that the prop is required. This doesn't really change anything, but it affects the output in `component-docs.json` which we use in Storybook.